input-device-capabilities/index.html

```diff
@@ -0,0 +1,36 @@
+---
+feature_name: Input Device Capabilities
+chrome_version: 47
+feature_id: 5681847971348480
+---
+
+<h3>Background</h3>
+<p>
+  The Input Device Capabilities API provides information about the capabilities of the physical
+  device responsible for generating an event.
+</p>
+<p>
+  This sample illustrates using <code>UIEvent.sourceCapabilities</code> (via a <code>click</code>
+  <code>UIEvent</code>) to obtain the capabilities of the device that triggered the click, and
+  then checks the <code>firesTouchEvents</code> property to determine whether that device can also
+  create a <a href="http://www.w3.org/TR/touch-events/#touchevent-interface"><code>TouchEvent</code></a>.
```

Comment:
We may want to add a comment like _This doesn't necessarily mean the device is a touch screen. For example, stylus and mouse devices typically generate touch events on mobile browsers._